modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -4,6 +4,8 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
+If you encounter errors in API Community Manager, refer to these explanations and instructions for recovery.
```

Comment:
"...while you are using..." ? So that it doesn't sound like ACM is at fault (even though it might be!)

Reply:
I like it! Made that change.